user-service/index.js

```diff
@@ -1,36 +1,42 @@
-import express from 'express'
-import cors from 'cors'
-import cookieParser from 'cookie-parser'
-import { URI_FRONTEND , PREFIX_LOGIN, PREFIX_CHANGE_PASSWORD, PREFIX_DELETE} from './common/config.js'
+import express from "express";
+import cors from "cors";
+import cookieParser from "cookie-parser";
+import {
+  URI_FRONTEND,
+  PREFIX_LOGIN,
+  PREFIX_CHANGE_PASSWORD,
+  PREFIX_DELETE,
+} from "./common/config.js";
 
-export const app = express()
-app.use(express.urlencoded({ extended: true }))
-app.use(express.json())
-app.use(cors({ credentials: true, origin: 'http://localhost' })) // config cors so that front-end can use
-app.options('*', cors())
-app.use(cookieParser())
+export const app = express();
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+app.use(cors({ credentials: true, origin: "http://localhost:3000" })); // for dev env

```

Comment:
I think can use environment variable to set this instead of manually changing it everytime.

`process.env.NODE_ENV === 'production' ? 'localhost' : 'localhost:3000'`